osgp/protocol-adapter-iec60870/osgp-protocol-adapter-iec60870/src/test/java/org/opensmartgridplatform/adapter/protocol/iec60870/domain/valueobjects/DomainTypeTest.java

```diff
@@ -0,0 +1,35 @@
+/**
+ * Copyright 2020 Smart Society Services B.V.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ */
+package org.opensmartgridplatform.adapter.protocol.iec60870.domain.valueobjects;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+import org.junit.jupiter.api.Test;
+
+class DomainTypeTest {
```

Comment:
I find it very hard to understand what this class is testing and why?
Perhaps there is some good reason to do this, but then I feel it is perhaps not clearly expressed in the test method names.

Same as with DomainInfoTest regarding the absence of public/private modifiers.

Reply:
I thought the DomainType would be a convenient class for the relation between DeviceType and DomainInfo. 
I think we could also use the domain info passed into the original connect request to route event messages to the correct domain. (With the assumption the correct domain info object is used in each connect request) Therefor I've removed the relation between DeviceType and DomainInfo (and thus the DomainType class). Added a DomainInfoFactory with a forDeviceType method only for testing.